SlicerCART/documentation/welcome.md

```diff
@@ -0,0 +1,35 @@
+# Welcome to SlicerCART Documentation
+Here is useful information for using SlicerCART in 3D Slicer.
+
+Please click on the following links to get started.
+
+* [Installation](installation.md)
+* [QuickStart](quickstart.md)
+* [Video Tutorials](videotutorials.md)
+* [User Guide](userguide.md) (contains detailed information)
+* [Developer Guide](../../CONTRIBUTING.md) (Contributing Guidelines)
+
+### Description
+
+SlicerCART is a 3D Slicer extension aimed to improve manual segmentation and classification 
+workflows across different teams. 
+
+More information about SlicerCART Purpose can be found [here](purpose.md).
+
+List of functionalities can be found [here](functionalities.md).
+
+**Keywords:** medical imaging, manual segmentation, manual correction, workflow, ground-truth segmentation, quality control
+
+**Abbreviations:**
+
+- MRI - Magnetic Resonance Imaging
+- CT - Computed Tomography
+- BIDS - Brain Imaging Data Structure
+- GUI - Graphical User Interface
+- QC - Quality Control
+
+
+### Other resources
+* [3D Slicer Tutorials](https://www.youtube.com/watch?v=QTEti9aY0vs&)
+* [3D Slicer Documentation](https://www.slicer.org/wiki/Documentation/Nightly/Training)
+
```

Comment:
People won't look at your Welcome.md they will read your README.md and go to the md files you cite in it. Would delete this file and put the information you judge primordial in README.md

Reply:
Ok Considered. Thanks for this input. You are right. However, when I go on successful slicer app (e.g. SlicerRT) there readme is very short and they refer to other documentation pages with more details. this is also in other slicer modules. Will keep the comment opened so others can give there input (I agree with you that this could be in the README as well)